openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -159,4 +159,6 @@ jQuery(function () {
     $(document).on('click', '.slide-toggle', function () {
         $(`#${$(this).attr('aria-controls')}`).slideToggle();
     });
+
+    $('#wikiselect').on('focus', function(){$(this).select();})
```

Comment:
`$(this).select()` should be `$(this).trigger('select')` but we can do this in a follow up.

Reply:
Should I follow up on this ?

Reply:
It's not necessary right now. The jQuery here is perfectly valid, it's just a little ambiguous, and personally, I don't like the shortcut versions such as `select` [1], `click` and `hover`.

Let's prioritize moving all the inline JS to the JavaScript files. Once we've done that, I think we should standardize on whether to use trigger or select and other shortcuts in our code!

[1] https://api.jquery.com/select/